frontend/src/metabase/nav/components/WhatsNewNotification/utils.unit.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { createMockVersionInfoRecord } from "metabase-types/api/mocks";
+import { getLatestEligibleReleaseNotes } from "./utils";
+
+describe("What's new - utils", () => {
```

Comment:
nit: we don't add an enclosing `describe` block and start with `describe` for what's being tested directly